test/metrics/gauge.spec.ts

```diff
@@ -10,6 +11,12 @@ describe("Gauge", function () {
   beforeEach(async function () {
     testingModule = await Test.createTestingModule({
       providers: [
+        {
+          provide: PROM_CONFIG,
+          useValue: {
+            prefix: ""
+          }
+        },
```

Comment:
This shouldn't need to be added if it's being setup correctly in the module

Reply:
Correct, just that this test does not provides the module, just the provider `makeGaugeProvider`, and it accepts an injection with provide of `PROMETHEUS_OPTIONS`.

Adding some tests regarding to that prefix being added to the name of the metric.